scripts/ci/travis/steps.sh

```diff
@@ -144,7 +144,7 @@ step_chainer_tests() {
 
 
 step_chainerx_python_tests() {
-    pytest "$REPO_DIR"/tests/chainerx_tests
+    pytest "$REPO_DIR"/tests/chainerx_tests -rf
```

Comment:
Let's add "error" and "xpass".

```suggestion
    pytest -rfEX "$REPO_DIR"/tests/chainerx_tests
```

Reply:
Sure will do.